modules/claims_api/app/swagger/v1/swagger.json

```diff
@@ -0,0 +1,7171 @@
+  },
+  "tags": [
+    {
+      "name": "Claims",
+      "description": "Allows authenticated and authorized users to access claims data for a single claim by ID, or for all claims based on Veteran data. No data is returned if the user is not authenticated and authorized.\n"
+    },
+    {
+      "name": "Disability",
+      "description": "Used for 526 claims."
+    },
+    {
+      "name": "Intent to File",
+      "description": "Used for 0966 submissions."
+    },
+    {
+      "name": "Power of Attorney",
+      "description": "Used for 21-22 and 21-22a form submissions."
+    }
+  ],
+  "components": {
+    "securitySchemes": {
+      "bearer_token": {
+        "type": "http",
+        "name": "token",
+        "scheme": "bearer",
+        "bearer_format": "JWT"
+      }
+    }
+  },
+  "paths": {
+    "/claims": {
+      "get": {
+        "summary": "Retrieves all claims for a Veteran",
+        "tags": [
+          "Claims"
+        ],
+        "operationId": "findClaims",
+        "security": [
+          {
+            "bearer_token": [
+
+            ]
+          }
+        ],
+        "description": "Uses the Veteran’s metadata in headers to retrieve all claims for that Veteran. An authenticated Veteran making a request with this endpoint will return their own claims, if any.",
+        "parameters": [
+          {
+            "in": "header",
+            "name": "X-VA-SSN",
+            "required": false,
+            "description": "Veteran SSN if consumer is representative",
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "in": "header",
+            "name": "X-VA-First-Name",
+            "required": false,
+            "description": "Veteran first name if consumer is representative",
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "in": "header",
+            "name": "X-VA-Last-Name",
+            "required": false,
+            "description": "Veteran last name if consumer is representative",
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "in": "header",
+            "name": "X-VA-Birth-Date",
+            "required": false,
+            "description": "Veteran birthdate if consumer is representative",
+            "schema": {
+              "type": "string"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "claim response",
```

Comment:
production has schema showing in the developer portal for responses but i dont see any for this rswag stuff, is schema for responses not possible with rswag?

Reply:
alright, well.... i found out what needs to be done to get schema to show up, but its not a quick and easy thing :(
don't want to commit what i have but you essentially have to define the schema for each response, either right in the spec or in the "components" section and reference them in the spec.

thats the bad news, we have to define structures for each response.
good news is that the tests do fail if the expected schema does not match what is in the response of the spec, so thats good at least.

https://github.com/rswag/rswag#getting-started
Step 3 shows an example of defining the schema within the spec itself

https://github.com/rswag/rswag#referenced-parameters-and-schema-definitions
shows an example of referencing a defined schema component.